src/styles/blocks/main/body.scss

```diff
@@ -0,0 +1,9 @@
+* {
```

Comment:
Using the universal selector (*) can lead to unexpected results and performance issues. It's generally better to apply styles to specific elements or classes.